config/jobs/kubernetes/sig-testing/integration.yaml

```diff
@@ -180,3 +218,44 @@ periodics:
         requests:
           cpu: 6
           memory: 20Gi
+- interval: 6h
```

Comment:
Interval open for debate...

Reply:
4 times a day is a good tradeoff, specially since we are going to iterate at the beginning to clean up the races